source/ruby.txt

```diff
@@ -0,0 +1,34 @@
+.. _ruby-language-center:
+
+===================
+MongoDB Ruby Driver
+===================
+
+.. toctree::
+   :titlesonly:
+
+   Ruby Driver  <https://www.mongodb.com/docs/ruby-driver/current/>
+   Ruby Mongoid ODM  <https://www.mongodb.com/docs/mongoid/current/>
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Introduction
```

Comment:
Suggestion:

While this might follow the structure of other pages, I found it slightly difficult to identify that the link in this section was for the driver and that the ODM would be in the next section because both were introduced here.

Maybe there could be a separate section after Introduction that mentions what you can do with the driver, similar to some of our driver docs landing pages.

E.g. (from docs/node, also similar to docs/java/sync)

> You can add the driver to your application to work with MongoDB in JavaScript. Download it using 
> [npm](https://www.npmjs.com/)
>  or set up a runnable project by following our Quick Start guide.
> 
> 
>